### ISSUE TITLE
Why not embrace open source?

### ISSUE THREAD

**Issue by shamilovtim** [CSSEGISandData/COVID-19#757]
This project has the ability to use the efforts of thousands of reliable, full time employed developers. In the past several days the project has been unreliable and plagued by technical issues. Why not allow open source contributions?

**Comment by yy** [comment]
Hey I've been also frustrated by it and create a repository with more transparent workflow / data: https://github.com/covid19-data/covid19-data Feel free to come join.

**Comment by ciscorucinski** [comment]
#558 Efforts are taking place here

**Comment by benjyz** [comment]
@yy great work, are there specific channels to organise opensource & data?

**Comment by tomtomme** [comment]
https://mshmedicalschoolhamburg-my.sharepoint.com/:f:/g/personal/thomas_langkamp_dozent_medicalschool-hamburg_de/EqlJuADXURFMmwu_m-gYVZEBRZ95o4taMz-OQePCDujvig?e=sc1VZG

This is my low-tech effort to analyze the chronology of mostly european contrys. I use Excel to easily get graphs to compare countries. So this does not replace any github csv raw data repo. But for the short term this is more useful to me. And everybody is welcome to join and contribute!

Background: Data is from multiple sources, mostly Jon Hopkins, but more aggregated, meaning only country level for Cina and US. For the last days other more up to date numbers like from Wikipedia and Berliner Morgenpost were merged and crosschecked. I had to transfer the numbers one by one. So this is completely manual. But with many hands this could be expanded to more countries easily. Comments are german for now. Feel free to contribute.
